pages/common/sonar-scanner.md

```diff
@@ -0,0 +1,20 @@
+# sonar-scanner
+
+> SonarScanner is a generic scanner for SonarQube for projects do not use any specific build tool like maven, gradle , etc.
+> More information: <https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/>.
+
+- Scan a project with configuration file in your project's root directory named `sonar-project.properties`:
+
+`sonar-scanner`
+
+- Scan a project using configuration file other than `sonar-project.properties`:
+
+`sonar-scanner -Dproject.settings={{myproject.properties}}`
```

Comment:
I think the whole key and value for this should be inside the token syntax, otherwise looks good to me. 👍🏻
```suggestion
`sonar-scanner -D{{project.settings=myproject.properties}}`
```